### PR TITLE
Test(runtime): repro of evm contract slow issue, bottleneck is on deser

### PR DESCRIPTION
**Pull request by ailisp**
This shows the slowness of deploy a common sized evm contract (the same test Illia did), Previous repro was a unit test: https://github.com/near/nearcore/compare/cache-slow-repro. This is rewritten with Aleksey's runtime standalone rust script facility which clearly show bottleneck is in deserialization:
```
running 1 test
      13.01µs get_key
      80.94ms compile_and_serialize_wasmer
        938.04µs run_method/instantiate
        4.06ms run_method/call
        949.55µs run_method/drop_instance
      6.00ms run_method
    89.79ms run_wasmer
  89.81ms run_vm

      3.18µs get_key
      19.49ms deserialize_wasmer
        951.93µs run_method/instantiate
        3.98ms run_method/call
        1.03ms run_method/drop_instance
      6.01ms run_method
    28.94ms run_wasmer
  28.96ms run_vm

      3.49µs get_key
      17.68ms deserialize_wasmer
        949.29µs run_method/instantiate
        3.99ms run_method/call
        710.68µs run_method/drop_instance
      5.69ms run_method
    26.24ms run_wasmer
  26.25ms run_vm

test script::evm_slow_deserialize_repro ... ok
```

It's improving in non-optimized wasmer1 (result below), and hopefully going to be fast enough in optimized wasmer1 (untested):
```
running 1 test
      13.57µs get_key
      49.76ms compile_and_serialize_wasmer1
        534.65µs run_method/instantiate
        2.38ms run_method/call
        4.16µs run_method/drop_instance
      2.96ms run_method
    53.52ms run_wasmer1
  53.55ms run_vm

      3.61µs get_key
      2.79ms deserialize_wasmer1
        364.05µs run_method/instantiate
        2.12ms run_method/call
        2.64µs run_method/drop_instance
      2.53ms run_method
    6.34ms run_wasmer1
  6.35ms run_vm

      2.99µs get_key
      1.80ms deserialize_wasmer1
        292.43µs run_method/instantiate
        2.10ms run_method/call
        2.65µs run_method/drop_instance
      2.44ms run_method
    5.32ms run_wasmer1
  5.33ms run_vm

test script::evm_slow_deserialize_repro ... ok
```

Test Plan
---------
```
cargo test --release --package near-vm-runner-standalone --bin near-vm-runner-standalone --features wasmer0_default,no_cache -- script::evm_slow_deserialize_repro --exact --nocapture

cargo test --release --package near-vm-runner-standalone --bin near-vm-runner-standalone --features wasmer1_default -- script::evm_slow_deserialize_repro --exact --nocapture

```

### CODE REVIEW

**Comment by matklad** [comment]
Where's `ZombieOwnership.bin` file?

**Comment by ailisp** [comment]
@matklad ah i forgot commit them :)

**Comment by ailisp** [comment]
Update numbers after #4165, both takes slightly less time (should be more accurate)

**Review by matklad — APPROVED** [review]

**Review by bowenwang1996 — COMMENTED** [review]
Is this PR supposed to be merged? Looks like it adds a test to reproduce some issue

**Comment by matklad** [comment]
Yes, this should be merged, it's a useful test case. I am not a code owner though, so my approval is not enough.

**Review by evgenykuzyakov — APPROVED** [review]

File: runtime/near-test-contracts/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 A collection of smart-contract used in nearcore tests.
 
 Rust contracts are build via `build.rs`, the Assembly Script contract is build manually and committed to the git repository.
+`res/near_evm.wasm` and `res/ZombieOwnership.bin` are taken from https://github.com/near/near-evm and it's for reproduce a
+performance issue encountered in evm contracts.
```
Comment (olonho) [inline runtime/near-test-contracts/README.md:5]: evm -> EVM.

**Review by olonho — COMMENTED** [review]
Committing binary blob without clear way to reproduce it doesn't look like a good idea. At the very least - please refer to exact git commit you used to build the contract, or commit the sourcecode somewhere in the folder.

**Comment by ailisp** [comment]
@olonho ptal :)

**Review by olonho — APPROVED** [review]

**Pull request merged**